src/plex_discord_rpc.gleam

```diff
@@ -1,14 +1,28 @@
 import argv
 import glint
 
+// Config
+import dot_env
+import util/env.{name_of_config_file}
+
 // Commands
 import commands/auth
+import commands/config
 import commands/help as help_command
 import commands/sample
 import commands/watch
 
 /// Entry point for the application
 pub fn main() {
+  // Init the config file
+  let _ = env.init_config_file()
```

Comment:
Unsure what happens if these errors and if we specifically need to handle it? `dot_env` itself should error anyway since we have 'ignore_missing_files' as False.